core/src/lib.rs

```diff
@@ -291,12 +340,29 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an symbolic representation of the schema suitable for applying across Mentat stores.
+    pub fn to_edn_value(&self) -> edn::Value {
+        edn::Value::Vector((&self.schema_map).into_iter()
+            .map(|(entid, attribute)| 
+                attribute.to_edn_value(self.get_ident(*entid)))
+            .collect::<Vec<_>>())
```

Comment:
I'm surprised you need a turbofish when you're collecting into an `edn::Value::Vector`. Did you need this because you had this in a `let`, then reorganized?